docs/sources/datasources/influxdb/configure-influxdb-data-source/_index.md

```diff
@@ -0,0 +1,260 @@
+---
+aliases:
+  - ../data-sources/influxdb/
+  - ../data-sources/influxdb/provision-influxdb/
+  - ../features/datasources/influxdb/
+  - provision-influxdb/
+description: Guide for using InfluxDB in Grafana
+keywords:
+  - grafana
+  - influxdb
+  - guide
+  - flux
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Configure the InfluxDB data source
+title: Configure the InfluxDB data source
+weight: 300
+refs:
+  build-dashboards:
+  provision-grafana:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/administration/provisioning/#provision-grafana
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana/<GRAFANA_VERSION>/administration/provisioning/#provision-grafana  
+---
+
+# Configure the InfluxDB data source
+
+This document describes how to configure the InfluxDB data source and reviews configuration options. 
+
+## Before you begin
+
+To configure the InfluxDB data source you must have the `Administrator` role.
+
+{{< admonition type="note" >}}
+Select the query language you want to use with InfluxDB before adding the InfluxDB data source. Configuration options differ based on query language type.
+{{< /admonition >}}
+
+InfluxData provides three query languages. Some key points to consider:
+
+- SQL is only available for InfluxDB v3.x.
+- Flux is a functional data scripting language for InfluxDB 2.x. Refer to [Query InfluxDB with Flux](https://docs.influxdata.com/influxdb/cloud/query-data/get-started/query-influxdb/) for a basic guide on working with Flux.
+- InfluxQL is SQL-like query language developed by InfluxData. It doesn't support more advanced functions such as JOINs.
```

Comment:
Nit: InfluxQL is a SQL-like query language...

Reply:
Also wondering if a list makes it more readable:

- **SQL**: Available only for InfluxDB v3.x.
- **Flux**: A functional data scripting language for InfluxDB 2.x. Refer to [Query InfluxDB with Flux](https://docs.influxdata.com/influxdb/cloud/query-data/get-started/query-influxdb/) for a guide on using Flux.
- **InfluxQL**: A SQL-like query language developed by InfluxData. It does not support advanced features like JOINs. For a comparison of Flux and InfluxQL, refer to [Flux vs InfluxQL](https://docs.influxdata.com/influxdb/v1.8/flux/flux-vs-influxql/) and [Why InfluxData Created Flux](https://www.influxdata.com/blog/why-were-building-flux-a-new-data-scripting-and-query-language/).

Reply:
I'd like to keep it as is, especially since there's a precondition - "Some points to consider". I feel full sentences flow better.